RF_ASM/Utils.cs

```diff
@@ -52,5 +52,13 @@ public static void DisplayHexDump(byte[] data)
                 Console.WriteLine();
             }
         }
+
+        public static void PadBytes(int desiredLength, byte[] ba)
+        {
+            while (ba.Length < desiredLength)
+            {
+                ba.Prepend((byte) 0);
```

Comment:
You cannot prepend arrays. In c# they are static, and so the prepend function here does nothing.

Reply:
The function is not used.